angular/src/lib/tabs/tab-header.component.ts

```diff
@@ -1,22 +1,18 @@
 /** @component tab */
 
-import { Component, OnInit, Input } from '@angular/core';
+import { Component, Input } from '@angular/core';
 
 @Component({
   selector: 'md-tab-header',
   template: `
     {{ heading }}
-    <div *ngIf="subHeading">{{ subHeading }}</div>
+    <div *ngIf="subHeading" class="md-tab-header--subheading">{{ subHeading }}</div>
```

Comment:
Would this make more sense to either be md-tab-header__subheading as it's an element?

Reply:
I do agree that the className here should be `md-tab-header__subheading`